### ISSUE TITLE
Error Packaging: "zlib_O2.bc: No such file or directory"

### ISSUE THREAD

**Issue by ilbeyataberkcan** [UnrealEngineHTML5/Documentation#70]
Hi everyone.

I was following the instructions at the "Detailed How To Build UE4 for HTML5". But whenever I tried to package my project i'm having this Error.

Exact error is 
`UATHelper: Packaging (HTML5):     shared:ERROR: D:\Unreal Engine HTML5 Fork\UnrealEngine\Engine\Platforms\HTML5\Source\ThirdParty\zlib\v1.2.8\lib-1.39.0-fc-mt\zlib_O2.bc: No such file or directory ("D:\Unreal Engine HTML5 Fork\UnrealEngine\Engine\Platforms\HTML5\Source\ThirdParty\zlib\v1.2.8\lib-1.39.0-fc-mt\zlib_O2.bc" was expected to be an input file, based on the commandline arguments provided)`

`PackagingResults: Error: D:\Unreal Engine HTML5 Fork\UnrealEngine\Engine\Platforms\HTML5\Source\ThirdParty\zlib\v1.2.8\lib-1.39.0-fc-mt\zlib_O2.bc: No such file or directory ("D:\Unreal Engine HTML5 Fork\UnrealEngine\Engine\Platforms\HTML5\Source\ThirdParty\zlib\v1.2.8\lib-1.39.0-fc-mt\zlib_O2.bc" was expected to be an input file, based on the commandline arguments provided)` 

I don't know what to do. I tried to do everything from scratch twice, but nothing changed. By the way when I went to the directory that error specifies I couldn't see any file; while when i check out the directory from the Epic Games Launcher version of the Engine there is files. 

Can you help me please. I'm going to have project that requires me to use this platform.

**Comment by ilbeyataberkcan** [comment]
Also when I try to rerun the HTML5Setup.sh, during the installation of zlib i have this error:

++ pwd
+ ZLIB_HTML5='/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/ThirdParty/zlib'
+ ZLIB_VERSION=v1.2.8
+ ZLIB_TAR_SRC='/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/../../../../Source/ThirdParty/zlib/v1.2.8/build'
+ ZLIB_TAR_NAME=zlib-1.2.8
+ ZLIB_HTML5_SRC='/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/ThirdParty/zlib/zlib-1.2.8'
+ ZLIB_HTML5_DST='/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/../../Source/ThirdParty/zlib/v1.2.8'
+ '[' '!' -d '/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/../../Source/ThirdParty/zlib/v1.2.8/include' ']'
+ '[' '!' -d '/d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Build/BatchFiles/../../Source/ThirdParty/zlib/v1.2.8/lib-1.39.0-fc' ']'
+ '[' '!' -d zlib-1.2.8 ']'
+ '[' 1 == 1 ']'
+ mkdir -p /d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Intermediate/1.39.0-fc/zlib-1.2.8
+ cd /d/Unreal Engine HTML5 Fork/UnrealEngine/Engine/Platforms/HTML5/Intermediate/1.39.0-fc/zlib-1.2.8
./build_html5_zlib.sh: line 77: cd: too many arguments

**Comment by ZeusLT** [comment]
same issue as in https://github.com/UnrealEngineHTML5/Documentation/issues/65 or https://github.com/UnrealEngineHTML5/Documentation/issues/5

try this but keep zlib instead of physx: https://github.com/UnrealEngineHTML5/Documentation/issues/5#issuecomment-653133458

Edit: actually sorry I missed something before, are you using linux? Someone in #65 topic had similar problem and he was using older toolchain version. He fixed it by removing a space in all folder names(because a lot of the scripts were missing quotes around the path), and upgraded to latest version (probably not necessary)

**Comment by ilbeyataberkcan** [comment]
First of all thank you for answer. I'm on Win10 64 bit.

I've seen the issue #5 but I haven't try it yet. I'm going to try it today. Also I'll be sure that there is no space in folder names; silly but actually it might be the problem. 😊

**Comment by ilbeyataberkcan** [comment]
Hi again @ZeusLT. I've tried many things and now I don't have the problem of zlib libraries. But, now i'm having trouble with PyhsX. And yes I've tried the solution in #5 but after each run i'm having trouble with a different part of PhysX. Should i keep trying until it works? Or should i delete everything and redo the steps?

Note: The running of the HTML5Setup.sh ends successfully. And i don't see any .bc file under ../Engine/Platforms/HTML5/Source/ThirdParty directory.
